sdk/spring/azure-spring-boot-starter-active-directory-b2c/README.md

```diff
@@ -116,6 +117,7 @@ application to use the Spring annotations and classes to protect the web app.
    | `azure.activedirectory.b2c.reply-url` | Contains one of the **Reply URL** from your application that you completed earlier. |
    | `azure.activedirectory.b2c.logout-success-url` | Specify the URL when your application logout successfully. |
    | `azure.activedirectory.b2c.user-flows` | Contains the name of the user flows that you completed earlier.
+   | `azure.activedirectory.b2c.user-name-attribute-name` | Contains the attribute name of the user name .
```

Comment:
How about we put this table under the section of `Key Concept`, just like what the aad starter does https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-starter-active-directory#configurable-properties.

Reply:
already done